torchtext/utils.py

```diff
@@ -6,11 +6,11 @@
 import zipfile
 
 import torch
+from iopath.common.file_io import file_lock
```

Comment:
This is the third time that iopath is being introduced as a dependency.
Did they become available on all platforms that torchtext is distributed?

* 1st
    https://github.com/pytorch/text/pull/1319
    https://github.com/pytorch/text/pull/1381
* 2nd
    https://github.com/pytorch/text/pull/1905

Reply:
Nope - will need to rework this to not depend on `iopath`.